.php_cs

```diff
@@ -1,29 +1,28 @@
 #!/usr/bin/env php
 <?php
-
-$finder = Symfony\CS\Finder::create()
-    ->ignoreDotFiles(true)
-    ->ignoreVCS(true)
-    ->exclude('app/config')
-    ->exclude('app/data')
-    ->exclude('app/Resources')
-    ->exclude('var')
-    ->exclude('vendor')
-    ->exclude('web/bundles')
-    ->exclude('web/css')
-    ->exclude('web/fonts')
-    ->exclude('web/js')
-    ->notPath('web/config.php')
-    ->in(__DIR__)
-;
-
-return Symfony\CS\Config::create()
-    ->level(Symfony\CS\FixerInterface::SYMFONY_LEVEL)
-    ->fixers([
-        '-psr0', // Ignore Tests\ namespace prefix mismatch with tests/ directory
-        'ordered_use',
-        'phpdoc_order',
-        'short_array_syntax',
+return PhpCsFixer\Config::create()
+    ->setRiskyAllowed(true)
+    ->setRules([
+        '@Symfony' => true,
+        '@Symfony:risky' => true,
+        'array_syntax' => ['syntax' => 'short'],
+        'ordered_imports' => true,
+        'phpdoc_order' => true,
```

Comment:
I only suggested it because that copyright is already present in all files, the idea is not for me to inject some new content, just enforce any updates and checks for consistent messaging.

I do understand that we might not want to have this, but if we don't why do we even have that header in all files?